BriefAssistant/ClientApp/src/app/components/welcomes/initial/initial.welcome.component.html

```diff
@@ -53,4 +53,5 @@ <h2>Welcome to the Brief Assistant for Initial Briefs!</h2>
         <a href="tel:+1-608-266-3440">608-266-3440</a>.
     </p>
 </div>
-<button class="btn btn-primary" type="submit" routerLink="/example" routerLinkActive="active">Continue to Example Brief</button>
+<button class="btn btn-primary" type="submit" routerLink="/initial-example" routerLinkActive="active">Continue to Example Brief</button>
+<router-outlet></router-outlet>
```

Comment:
The router outlet is not needed here. The only time you would need it is if you are dynamically displaying another component inside of this one based on the current route. Currently this is only true in app.component.